guvnor-asset-mgmt/guvnor-asset-mgmt-client/src/test/java/org/guvnor/asset/management/client/editors/project/structure/ReleaseRepositoryPresenterTest.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2014 JBoss Inc
```

Comment:
Hehe I might have copied from another class or copy+refactor another class, you are right for this one.